issue-wanted.cabal

```diff
@@ -1,9 +1,9 @@
-cabal-version:       2.2
+cabal-version:       2.4
 name:                issue-wanted
 version:             0.0.0
-description:         Web application to help beginners to start contributing into Haskell projects
+description:         Please see the README on Github at <https://github.com/kowainik/issue-wanted#readme>
```

Comment:
No need to change the description on the link to README. If you feel that it's not very informative we could change it to something more useful, what do you think?

Reply:
Got it. I think the old description is fine.